src/python_routing_v02/diffusive_utils.py

```diff
@@ -0,0 +1,654 @@
+import os
+import sys
+import numpy as np
+import pandas as pd
+from itertools import chain, islice
+import itertools
+from functools import partial, reduce
+
+import troute.nhd_network as nhd_network
+#from pyuniflowtzlt import uniflow_lookuptable
+
+def adj_alt1(mx_jorder_tw
+        , ordered_reaches
+        , geo_cols
+        , geo_index
+        , geo_data
+        , dbfksegID
+        , z_all   
+        ):  
+
+    # Adjust altitude so that altitude of the last node (=fake segment) of a reach is equal to that of
```

Comment:
This entire module should has a lot of functions with comments that seem to describe the functions, they should probably be refactored as [pep-257](https://www.python.org/dev/peps/pep-0257/) doc strings